express-app/src/index.js

```diff
@@ -0,0 +1,176 @@
+const express = require('express');
```

Comment:
As discussed this needs to be broken up into multiple folders/files/functions.

Reply:
Done